google/cloud/bigquery_storage_v1/reader.py

```diff
@@ -153,7 +153,7 @@ def __iter__(self):
     def _reconnect(self):
         """Reconnect to the ReadRows stream using the most recent offset."""
         self._wrapped = self._client.read_rows(
-            self._name, self._offset, **self._read_rows_kwargs
+            self._name, offset=self._offset, **self._read_rows_kwargs
```

Comment:
I recall `self._client` is the generated client. That means we should be using `read_stream=self._name` here, right?

Reply:
Checked the code and you're right, I will adjust. Nice catch!